.github/workflows/pending-clear.yml

```diff
@@ -0,0 +1,30 @@
+name: "clear pending status"
```

Comment:
I wonder if this workflow file could delete the `set pending status / action (pull_request_target)` run and maybe itself from the list of displayed action.
It would just look a bit cleaner.

Reply:
Deleting a status doesn't seem to be possible.